src/client/pages/ToeristischeVerhuur/ToeristischeVerhuur.tsx

```diff
@@ -193,19 +193,25 @@ export default function ToeristischeVerhuur() {
                   </>
                 )}
               </Heading>
-              <p className={styles.DetailText}>
-                Het aantal resterende nachten is gebaseerd op uw meldingen voor
```

Comment:
Moet deze tekst niet verschijnen voor `> 2021` ?

Reply:
Ja klopt de oude tekst kunnen we dan idd het best gebruiken!